src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import model.Operation;
+import service.OperationStrategy;
+import service.ReportMaking;
+import service.Reporter;
+import serviceimpl.FruitPackerImpl;
+import serviceimpl.OperationStrategyImpl;
+import serviceimpl.ReaderImpl;
+import serviceimpl.ReportMakingImpl;
+import serviceimpl.ReporterImpl;
+import serviceimpl.WriterImpl;
+import strategy.BalanceOperation;
+import strategy.Operating;
+import strategy.PurchaseOperation;
+import strategy.ReturnOperation;
+import strategy.SupplyOperation;
+
+public class Main {
+    public static final String SOURCE_FILE_NAME = "src/main/resources/sourceFile.csv";
+    public static final String RESULT_FILE_NAME = "src/main/resources/resultFile.csv";
+    private static final Map<Operation, Operating> operationsMap = new HashMap<>();
```

Comment:
Let's use `Map.of()` here.
https://www.geeksforgeeks.org/initialize-a-static-map-using-java-9-map-of/